src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -90,18 +122,29 @@ public void resetEncoders(){
 
   // TODO 2.2.2: Implement these 4 methods
   public double getLeftPosition(){
-    return 0;
+    if (RobotBase.isReal()){
+      return leftMotor1.getEncoder().getPosition();
+    }else{
```

Comment:
For the real robot, you need to divide by the gear ratio and multiply by the wheel circumference. Do not change the sim version, only the one for the real robot.

Reply:
You  need to do the same thing for the right motor.